bin/lang-js/src/index.ts

```diff
@@ -39,15 +37,6 @@ async function main() {
       `timeout for a function execution in seconds (default: ${defaultTimeout})`,
       { default: defaultTimeout },
     )
-    .arguments("<kind:string>")
```

Comment:
Yep, that was the tiny change that makes all other changes necessary! Doesn't look as bad as I thought it might